lib/utilities.js

```diff
@@ -4,11 +4,11 @@ const { basename } = require('path');
 
 const { between } = require('./strings');
 
-const deprecate = (
-  // Wrap method to mark it as deprecated
-  fn // function (optional)
-  // Returns: function, wrapped with deprecation warning
-) => {
+// Wrap method to mark it as deprecated
+//   fn - <function>
```

Comment:
I propose to change it to `Function` here and everywhere else (similar to how it's done in Node.js docs):
```suggestion
//   fn - <Function>
```

Reply:
I like this idea, though it will require to change [types](https://github.com/metarhia/metaschema/blob/master/lib/types.js) in metaschema in order to properly parse parameters types. Another option is to to convert them to lowercase, but this doesn't look great to me.